tests/test_items.py

```diff
@@ -0,0 +1,26 @@
+import mock
+import unittest
+from click import ClickException
+from shub import items
+
+
+class ItemsTest(unittest.TestCase):
```

Comment:
It would be a little better if these tests used click.testing.CliRunner. It's more end-to-end (more coverage).

It wouldn't change much and we'd be sure the interfaces are running fine :)